plugins/inputs/kapacitor/README.md

```diff
@@ -23,12 +23,20 @@ The Kapacitor plugin will collect metrics from the given Kapacitor instances.
   # insecure_skip_verify = false
 ```
 
-### Measurements & Fields
+### Measurements and fields
 
 - [kapacitor](#kapacitor)
     - [num_enabled_tasks](#num_enabled_tasks) _(integer)_
     - [num_subscriptions](#num_subscriptions) _(integer)_
     - [num_tasks](#num_tasks) _(integer)_
+- [kapacitor_alert](#kapacitor_alert)
+	- [notification_dropped](#notification_dropped) _(integer)_
+	- [primary-handle-count](#primary-handle-count) _(integer)_
+	- [secondary-handle-count](#secondary-handle-count) _(integer)_
+- [kapacitor_cluster](#kapacitor_cluster) _(integer)_ (Kapacitor Enterprise only)
+	- [dropped_member_events](#dropped_member_events) _(integer)_
+	- [dropped_user_events](#dropped_user_events) _(integer)_
+	- [query_handler_errors](#query_handler_errors) _(integer)_
```

Comment:
All of these metrics only appear in the Enterprise edition of Kapacitor so we should call that out somewhere.